ingestion/functions/make_rules.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+# Need to run this from the directory containing this script, aws.py, and define_rules.py
+# Run define_rules.py first to generate `rules.json`, then run this to create the rules
+
+import json
+import subprocess
+import sys
+
+
+FILE_NAME = "rules.json"
+
+
+print(f"Reading source information from {FILE_NAME} to create AWS EventBridge rules")
+
+with open(FILE_NAME, "r") as f:
+    rules = json.load(f)
+    for rule in rules:
+        rule_name = rule.get("rule_name")
+        target_name = rule.get("target_name")
+        source_name = rule.get("source_name")
+        job_name = rule.get("job_name")
+        if not rule_name:
+            print(f"Missing rule name in rule {rule}")
+            sys.exit(1)
+        if not target_name:
+            print(f"Missing target name in rule {rule}")
+            sys.exit(1)
+        if not source_name:
+            print(f"Missing source name in rule {rule}")
+            sys.exit(1)
+        if not job_name:
+            print(f"Missing job name in rule {rule}")
+            sys.exit(1)
+        cmd = f"./aws.py schedule {rule_name} -t {target_name} -s \"{source_name}\" -j {job_name}"
```

Comment:
I would try to import and call out, but aws.py isn't implemented that way, so not a blocker

Reply:
Maybe better down the road to put `define_rules.py` and `make_rules.py` functions inside `aws.py`?
Going to leave it alone for now.

Reply:
Yes, good idea for strongcat.